### PR TITLE
updates in CRUD operation section.

### PR DESCRIPTION
**Pull request by bparth24**
@pleerock @AlexMesser cc: @Therecanbeonlyone1969 

Please review the updates. These updates are based on the comments from Manu Sporny here is the [link](https://github.com/w3c/did-spec-registries/pull/556#pullrequestreview-2028725506)

Note: Once we merge this PR - I will comment and reply back to Manu that we have updated the specification for details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,13 @@ Requirements:
 - The provided DID MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how create operation works:
+
+- The create DID request is parsed by the ITN agent, which validates its syntax as well as the value of each field to follow specific rules.
+- The ITN agent generates the DID identifier based on the `did:itn` method specification, then the network stores the Hash(DID) and Hash(DID Document) in a smart contract on all the utilized ledger (one private and one public)
```
Comment (AlexMesser) [inline README.md]: DID identifier is generated by the sender, not by ITN Agent.

File: README.md [hunk 2]
```diff
@@ -126,6 +133,10 @@ Requirements:
 - A DID string MUST be provided.
 - The provided DID MUST be compliant with the `did:itn` method.
 
+Here is how resolve operation works:
+
+- The ITN Agent validates the provided DID, retrieves the DID Document from the federated content-addressable storage solution, retrieves the DID Document proof from one or more utilized ledger (either private or public) and returns the DID Document along with the DID Document proof to the requestor.
```
Comment (AlexMesser) [inline README.md]: ITN Agent is not involved in resolution process. The request goes directly to the DID Resolver.

File: README.md [hunk 3]
```diff
@@ -137,6 +148,10 @@ Requirements:
 - The provided DID string MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how update operation works:
+
+- The ITN agent is responsible to verify that the initiator of the update request is the controller of the DID subject that will be updated. The authentication method defined in the DID Document will be used to authenticate the controller.
```
Comment (AlexMesser) [inline README.md]: Update request sender must be specified as a DID Document `controller` and is authenticated by verifying the update request signature. It must be signed by the recovery key which is used before to generate the DID identifier.

Comment (AlexMesser) [reply README.md]: Also we need to mention that DID Document is updated in CAS and in ledgers.

File: README.md [hunk 4]
```diff
@@ -149,6 +164,10 @@ Requirements:
 - The provided DID string MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how revoke operation works:
+
+- The ITN agent is responsible to verify that the initiator of the revoke request is the controller of the DID subject that will be revoked. The authentication method defined in the DID Document will be used to authenticate the controller. Then ITN agent perform the revoke DID operation.
+
```
Comment (AlexMesser) [inline README.md:170]: Same as Update DID flow

File: README.md [hunk 5]
```diff
@@ -161,6 +180,10 @@ Requirements:
 - The provided DID string MUST be compliant with the `did:itn` method.
 - The provided DID document MUST be compliant with the DID V1.0 DID document specification.
 
+Here is how recover operation works:
+
+- The ITN agent is responsible to verify that the initiator of the recover request is the controller of the DID subject that will be recovered. The authentication method defined in the DID Document will be used to authenticate the controller. Then ITN agent perform the recover DID operation.
+
```
Comment (AlexMesser) [inline README.md:186]: Same as Update DID flow

File: README.md (hunk 1 above)
Comment (bparth24) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (bparth24) [reply README.md]: Updated.

File: README.md (hunk 3 above)
Comment (bparth24) [reply README.md]: Updated.

File: README.md (hunk 4 above)
Comment (bparth24) [reply README.md:170]: Updated.

File: README.md (hunk 5 above)
Comment (bparth24) [reply README.md:186]: Updated.

**Review by pleerock — APPROVED** [review]

**Pull request merged**